utxo/utxo.go

```diff
@@ -740,7 +739,9 @@ func (uv *UtxoVM) PreExec(req *pb.InvokeRPCRequest, hd *global.XContext) (*pb.In
 			// FIXME zq @icexin need to return contract not found error
 			uv.xlog.Error("PreExec NewContext error", "error", err,
 				"contractName", tmpReq.GetContractName())
-			if i < len(reservedRequests) && err.Error() == "Key not found" {
+			if i < len(reservedRequests) && strings.HasSuffix(err.Error(), "not found") {
+				request := *tmpReq
```

Comment:
why not using tmpReq direct to append into request?

Reply:
because  the last `tmpReq` will overwrite the ahead